components/Testimonial.js

```diff
@@ -1,35 +1,72 @@
-import Paragraph from "./typography/Paragraph";
+import { FaArrowLeft, FaArrowRight } from 'react-icons/fa';
```

Comment:
I prefer not to import new library for the icons, better you should use `components/icons/ArrowLeft.js` or extend them to use it inside the component. Kindly remove the new library you imported in package files.